x/staking/keeper/unbonding.go

```diff
@@ -127,17 +125,7 @@ func (k Keeper) GetValidatorByUnbondingID(ctx context.Context, id uint64) (val t
 		return types.Validator{}, types.ErrNoValidatorFound
 	}
 
-	value, err := store.Get(valKey)
-	if err != nil {
-		return types.Validator{}, err
-	}
-
-	if value == nil {
```

Comment:
is this still a valid case or collections returns error?

Reply:
It returns error from collections when validator key is not found